_faq/application_template.md

```diff
@@ -4,6 +4,8 @@ question: "Do you have an application template?"
 coala 2017 Application Template
 ===============================
 
+#### Note: We encourage you to use [Username Checker](https://username-check.herokuapp.com/) to check if your username is proper to use.
```

Comment:
Needs to be stronger!

Make them put a link with their username into the template, e.g. at "GitHub username"

i.e. https://username-check.herokuapp.com/?username=your_username